src/Microsoft.Azure.WebJobs.Host/Dispatch/DummyDispatchQueueHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Host.Dispatch
+{
+    // this class is used for local testing
+    // an extension developer can test DispatchQueueHandler without 
+    // providing an azure storage account
+    internal class DummyDispatchQueueHandler : IDispatchQueueHandler
+    {
+        private IMessageHandler _messageHandler;
+        internal DummyDispatchQueueHandler(IMessageHandler messageHandler)
+        {
+            _messageHandler = messageHandler;
+        }
+
+        // any messages enqueued will get processed in worker pool immediately
+        public Task EnqueueAsync(JObject message, CancellationToken cancellationToken)
+        {
+            // start executing the function
+            // ignore the function result, no retry or poisonQueue
+            _messageHandler.TryExecuteAsync(message, cancellationToken);
```

Comment:
>_messageHandler [](start = 12, length = 15)

Nit: 
Task taskIngore = _messageHandler....

(I thought we issued a warning on non-awaited tasks) 
Makes it obvious that there is indeed a task, but we're not blocking on it.  
I think this has similar semantics to the live queue.